script/ccip/WaypointTransfer.s.sol

```diff
@@ -26,18 +26,19 @@ contract CCIPWaypointTransfer is Script {
             receiver: vm.envAddress("CCIP_RECEIVER")
         });
 
-        uint256 amount = 10 * 10 ** cfg.usdPlus.decimals();
+        uint256 amount = 1499955;
```

Comment:
any comment to understand why this value ?

Reply:
no special value, just how much I was moving